instructure/canvas-lms
# Peer review csv export

I have added a simple CSV export functionnality to the Peer review page. 

The button to export the CSV has this translation key: "download_csv"

The downloaded CSV has the following information: 

Student sortable name | Student ID | Student Submission ID | Current Grade | Grade 1 | Peer 1 ID | Grade 2 | Peer 2 ID ... 

I think it is a very usefull functionnality if someone wants to do some statistics or calculations based on peer reviews. 

test-plan:
* create a course with at least one assignment by activating the "peer_review" functionnality
* add two students to this course
* masquerade a student and fill in the assignment
* assign the peer_review to the other student
* masquerade the other student and achieve the peer review
* once it's done, go to the "peer-reviews" page and click on the "download CSV" button
* check the CSV, it should have 6 Columns: 
   * student sortable name, 
   * student id, 
   * submission id, 
   * current grade, which should only be "-" as the submission has no score yet
   * grade 1 - the grade given by the first peer reviewer, 
   * id of the first peer reviewer 
* you can add more students to the course and do more peer reviews, especially do 2 peer reviews for one student to see if the "grade 2" column is properly created. 

I hope it'll help the community!
